### PR TITLE
Move user dropdown options to modal

### LINKED ISSUE #53: Brak opcji przełączenia na moduł wiadomości w usprawnionym interfejsie mobilnym.
Interfejs mobilny z PWA to ciekawa alternatywa dla aplikacji w tych czasach zwiększająca wygodę użytkowania na smartfonach (zwłaszcza w ramach zwykłego konta - bez jakiejkolwiek aplikacji oficjalnej).
Jednak w tym kombo brakuje jednej z kluczowej dla niektórych funkcji, którą posiadał Wulkanowy, jest dostępna poprzez wersje www i ma być płatna dla oficjalnej apki eduVULCAN - dostęp do modułu wiadomości.

Uważam, że taki przycisk może się znaleźć na górnym pasku strony (tam, gdzie logo Dziennika VULCAN/eduVULCAN) po prawej stronie.
Jeśli chodzi o liczbę nieodczytanych wiadomości to jest to dość proste (jeśli nie będzie to przycisk przeniesiony z oryginalnego menu), gdyż API do tego obsługi, jako jedno z nielicznych, posiada stały adres poboru danych: https://dziennik-uczen.vulcan.net.pl/[szkoła/miasto/gmina/powiat...]/api/WiadomosciNieodczytane.

### LINKED ISSUE #61: Sometimes patch's js code doesn't run
When you open `https://dziennik-uczen.vulcan.net.pl/warszawazoliborz/App/.../tablica` it works normally, but `https://dziennik-uczen.vulcan.net.pl/warszawazoliborz/App/.../` does look like on the screenshot
image

### PR DESCRIPTION
**Pull request by PanLiszka**
Closes #53
Fixes #61

### CODE REVIEW

File: apply.js [hunk 1]
```diff
@@ -9,215 +9,224 @@
...
-      css: ["alignDetailedGradesButton.css"],
+    {
+      name: "Align Detailed Grades Button",
+      description: "todo",
+      files: {
+        css: ["alignDetailedGradesButton.css"],
+      },
+      allowedHostsCss: ["uczen.eduvulcan.pl", "dziennik-uczen.vulcan.net.pl"],
     },
-    allowedHostsCss: ["uczen.eduvulcan.pl", "dziennik-uczen.vulcan.net.pl"],
```
Comment (banocean) [inline apply.js:33]: I don't think this change was supposed to be there

File: patches/moveUserOptionsToHeader/script.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            : document.querySelector(".side_important-text.side_student")
+                  ?.textContent,
+        username: document.querySelector(".user div:nth-child(2)").lastChild
+            .textContent,
+    };
+
+    document.querySelector(".user").click();
+    const userLinks = document.querySelectorAll(".user__links a");
+
+    /* Close Aside */
```
Comment (banocean) [inline patches/moveUserOptionsToHeader/script.js:28]: tbh code here is self-explanatory

File: patches/moveUserOptionsToHeader/script.js
Comment (banocean) [inline patches/moveUserOptionsToHeader/script.js:1]: since there is no action added to the links, it refreshes the website

**Review by banocean — CHANGES_REQUESTED** [review]
image
this doesn't look like it should

**Review by banocean — CHANGES_REQUESTED** [review]
Breaks bottom navigation

File: patches.json
Comment (banocean) [inline patches.json:1]: This for sure is out of date and only doesn't conflict because everything is overwritten

**Review by banocean — CHANGES_REQUESTED** [review]
formatting is missing

**Review by Marioneq4958 — DISMISSED** [review]
It looks good, I'll test it immediately.

**Review by banocean — APPROVED** [review]

**Review by rotkiW285 — APPROVED** [review]
LGTM

**Review by rotkiW285 — APPROVED** [review]

File: patches/moveUserOptionsToHeader/styles.css [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    .header__logo-product > img {
+        width: auto !important;
+    }
+
+    header.app__header {
+        height: 60px;
+    }
+
+    body.blue .user-avatar {
+        background: #38817f !important;
```
Comment (PanLiszka) [inline patches/moveUserOptionsToHeader/styles.css:107]: opera_MuXJ8p2wOT

Text on avatar should be white

Comment (PanLiszka) [inline patches/moveUserOptionsToHeader/styles.css:107]:
```suggestion
        background: #38817f !important;
        color: #ffffff;
```

**Review by banocean — APPROVED** [review]

**Pull request merged**